src/web/mjs/connectors/Mangajp.mjs

```diff
@@ -0,0 +1,18 @@
+import WordPressMangastream from './templates/WordPressMangastream.mjs';
+
+export default class Mangajp extends WordPressMangastream {
+    constructor() {
+        super();
+        super.id = 'mangajp';
+        super.label = '漫画RAW(mangajp)';
+        this.tags = ['manga', 'japanese'];
+        this.url = 'https://mangajp.top';
+        this.path = '/manga/list-mode/';
+        this.requestOptions.headers.set('x-referer', 'https://mangajp.top/');
```

Comment:
Does the referer absolutely need the last slash? cause we could need this.url instead .